doc/developer-guides/hld/hld-APL_GVT-g.rst

```diff
@@ -311,18 +311,18 @@ load and run-time phases ranging in tens of thousands per second.
 
    Access Patterns of Running 3D Workloads
 
-High Level Architecture
+High-Level Architecture
 ***********************
 
 :numref:`gvt-arch` shows the overall architecture of GVT-g, based on the
 ACRN hypervisor, with Service VM as the privileged VM, and multiple user
-guests. A GVT-g device model working with the ACRN hypervisor,
+guests. A GVT-g device model working with the ACRN hypervisor
 implements the policies of trap and passthrough. Each guest runs the
 native graphics driver and can directly access performance-critical
 resources: the Frame Buffer and Command Buffer, with resource
-partitioning (as presented later). To protect privileged resources, that
-is, the I/O registers and PTEs, corresponding accesses from the graphics
-driver in user VMs are trapped and forwarded to the GVT device model in
+partitioning (as presented later). To protect privileged resources—that
```

Comment:
We do not need spaces around the hyphen?

Reply:
Don't need spaces around an em dash. (Some people use the narrower en dash with spaces around it.)